package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-json": "^1.2.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.6.1",
+    "fs": "0.0.1-security",
```

Comment:
Remove this lib - `fs` is a native JS lib.

Reply:
Will do